hadoop-hdds/server-scm/src/main/java/org/apache/hadoop/hdds/scm/pipeline/PipelinePlacementPolicy.java

```diff
@@ -110,17 +97,11 @@ boolean meetCriteria(DatanodeDetails datanodeDetails, int nodesRequired) {
         pipelineNumDeductable++;
       }
     }
-    boolean meet = (nodeManager.getPipelinesCount(datanodeDetails)
-        - pipelineNumDeductable) < heavyNodeCriteria;
-    if (!meet && LOG.isDebugEnabled()) {
-      LOG.debug("Pipeline Placement: can't place more pipeline on heavy " +
-          "datanode： " + datanodeDetails.getUuid().toString() +
-          " Heaviness: " + nodeManager.getPipelinesCount(datanodeDetails) +
-          " limit: " + heavyNodeCriteria);
-    }
-    return meet;
+    return pipelines.size() - pipelineNumDeductable;
   }
 
+
+
   /**
    * Filter out viable nodes based on
    * 1. nodes that are healthy
```

Comment:
We should add a note to this Java doc that the returned list is sorted in order of pipeline count, starting with the lowest to highest.

Reply:
Updated.